java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/cp/FencedLockTest.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hazelcast.simulator.tests.cp;
+
+import com.hazelcast.cp.lock.FencedLock;
+import com.hazelcast.simulator.hz.HazelcastTest;
+import com.hazelcast.simulator.test.BaseThreadState;
+import com.hazelcast.simulator.test.annotations.AfterRun;
+import com.hazelcast.simulator.test.annotations.Setup;
+import com.hazelcast.simulator.test.annotations.TimeStep;
+import com.hazelcast.simulator.test.annotations.Verify;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicLong;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Acq/release for uncontended and contented locks. Only a single thread per-client should be used. Multiple clients are fine.
+ * This is because the uncontended locks are locks that I want to have no contention on.
+ * <p>
+ * Default is to always acquire-release uncontended locks.
+ * <p>
+ * Default number of uncontended locks is 1000; this can be overridden by defining the property 'uncontendedLockCount'.
+ * Default set of contended locks is 10. You can override this by defining the property 'contendedLockNames' in your configuration.
+ */
+public class FencedLockTest extends HazelcastTest {
+    public List<String> contendedLockNames =
```

Comment:
Would it not make more sense to have a 'contendedLockCount'? This way the number of contended locks is configurable.

Reply:
It make sense -- though, I've been thinking recently this contended scenario is most likely useless.

Reply:
Done.